common/src/main/java/com/wynntils/models/wynnitem/AbstractItemInfoDeserializer.java

```diff
@@ -416,8 +393,8 @@ protected List<Pair<StatType, StatPossibleValues>> parseVariableStats(JsonObject
             }
             // Range will always be stored such as "low" means "worst possible value" and
             // "high" means "best possible value".
-            RangedValue range = StatCalculator.calculatePossibleValuesRange(
-                    baseValue, preIdentifiedItem, statType.showAsInverted());
+            RangedValue range =
+                    StatCalculator.calculatePossibleValuesRange(baseValue, preIdentified, statType.showAsInverted());
```

Comment:
This is actually a bugfix for a typo, I already fixed this either in the gear encoding or the charm PR, but it is really obviously broken in this case, so I promoted the fix here.